Project.toml

```
@@ -21,7 +21,10 @@ SimpleUnPack = "ce78b400-467f-4804-87d8-8f486da07d0a"
 ArrayInterface = "6, 7"
 DataStructures = "0.18"
 DiffEqBase = "6.122"
+LinearAlgebra = "1"
+Logging = "1"
```

There's an unfortunate problem (https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958#update-november-9th-2023-2) that means it is generally safer to use
```suggestion
LinearAlgebra = "<0.0.1, 1"
Logging = "<0.0.1, 1"
```